PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionContentPanel.kt

```diff
@@ -114,6 +121,9 @@ open class SparkSubmissionContentPanel(private val myProject: Project, val type:
             // Don't add more we won't like to add more message labels
     )
 
+    private var fileSystem: ADLSGen2FileSystem? = null
+    private var rootPath: String? = null
```

Comment:
Only `rootPath` might be enough

Reply:
Or can it read from `root path` field?

Reply:
it gets from subscribing  uploadFieldSubject 

---
In reply to: [285430221](https://github.com/microsoft/azure-tools-for-java/pull/3124#discussion_r285430221) [](ancestors = 285430221)

Reply:
?

---
In reply to: [285430079](https://github.com/microsoft/azure-tools-for-java/pull/3124#discussion_r285430079) [](ancestors = 285430079)